packages/backend/src/modules/ApplicationModule.ts

```diff
@@ -8,6 +8,6 @@ export interface ApplicationModule {
 }
 
 export interface TvlSubmodule {
-  updaters?: AssetUpdater[]
+  assetUpdaters?: AssetUpdater[]
```

Comment:
Maybe it should be called "reportUpdaters" and the interface should also change to "ReportUpdater"